README.md

```diff
@@ -0,0 +1,78 @@
+# Ruby on Rails Grannys Recipe Team project
+
+## Live Demo
+
```

Comment:
- [ ] Kindly fill these sections to make a professional readme file. Making a professional readme file is a general requirement for all projects.